### PR TITLE
simply pytorch install by using pytorch-nightly instead of source

### PR DESCRIPTION
**Pull request by soumith**
simply pytorch install by using pytorch-nightly instead of source

### CODE REVIEW

**Comment by jmp84** [comment]
Patching and trying instructions again (or did you already try?).

**Comment by soumith** [comment]
I haven't tried.

**Comment by soumith** [comment]
if you are using cuda9, it is: `conda install pytorch-nightly cuda90 -c pytorch`

File: README.md [hunk 1]
```diff
@@ -30,37 +30,9 @@ environment with Python 3.6, you can install one via [Miniconda3](https://conda.
...
-  pushd pytorch
-  # Pin to a specific commit for now to guard against breakage in HEAD.
-  git checkout d154d32
-  git submodule update --init
-  NCCL_ROOT_DIR="${NCCL_ROOT_DIR}" python3 setup.py install \
-    2>&1 | tee PYTORCH_OUT
+  conda install pytorch-nightly -c pytorch
   ```
 
 - Build [Caffe2](http://caffe2.ai/) from source (under PyTorch):
```
Comment (theweiho) [inline README.md]: I think the main reason that we reverted to building PyTorch from source was b/c @jhcross found some incompatibility between the nightly builds of Caffe2/PyTorch and ONNX? Has that been resolved? (@jhcross - can you elaborate what the issue was again?)

**Comment by theweiho** [comment]
You can also try seeing if 
```
conda uninstall -y caffe2 caffe2-cuda8.0-cudnn7
conda install -y -c caffe2 caffe2-cuda8.0-cudnn7
```
before trying to install ONNX helps

**Comment by jmp84** [comment]
@theweiho, same issue.

**Comment by pjh5** [comment]
@jmp84 where's the segfault?
pytorch-nightly does not include Caffe2
It is safe to clone Pytorch and only build Caffe2 from it and get the Pytorch installation from conda; this will save all the time needed to build Pytorch. You can't do this with the Caffe2 conda packages because they are not built with Aten. The pytorch-caffe2_cuda8.0_cudnn7 builds in the caffe2 conda channel are built with Aten, and also include Pytorch, but the Pytorch included from these builds will be slightly different than the one from pytorch-nightly (though these differences should not affect you)

**Comment by theweiho** [comment]
@anderspapitto @bddppq - James Cross mentioned y'all helped him debug the ONNX compatibility/segfault issue - and that it may have been due to differences in gcc compiler versions or something?

**Comment by anderspapitto** [comment]
we got something to work, but there wasn't full clarity into exactly what the issue had been.

If you're able to make changes and things work, then you're probably fine - the crashes we were running into were very upfront

**Comment by jmp84** [comment]
Fixed in https://github.com/pytorch/translate/commit/7ba9dfcb877c645e8f8f91bfca6ab6ab77dc8ca3 by @theweiho

**Pull request closed without merging**